### DISCUSSION TITLE
Looking for release 1.8.2

### DISCUSSION THREAD

**Discussion by alperencanesen** [jodli/CreativeMod#50] [Q&A]
Hello,

I’m an experienced player with modded game files, and I’m currently looking for the 1.8.2 version of Creative Mod to continue playing. I had it previously, but unfortunately, I formatted my entire disk without backing it up. Having access to the 1.8.2 release would significantly improve my gaming experience.

Thank you in advance!

**Comment by alperencanesen** [comment]
I'm actually looking for game version 1.1.96 release

**Comment by jodli** [comment]
Hey, all downloads are contained on the [mod portal page](https://mods.factorio.com/mod/creative-mod/downloads), including previous downloads of older versions.

1.8.2 was the last version for Factorio 1.1. Is this what you're looking for?

**Comment by alperencanesen** [comment]
Downloaded the alter versions from the modpage
